CwdUtils.c

```diff
@@ -0,0 +1,158 @@
+#include "config.h" // IWYU pragma: keep
+
+#include "CwdUtils.h"
+
+#include <stdbool.h>
+#include <stdlib.h>
+#include <wchar.h>
+
+#include "XUtils.h"
+
```

Comment:
Second blank line after the header includes.

Reply:
Done.